### PR TITLE
Add Privacy Policy

### PR DESCRIPTION
**Pull request by wrussell1999**
Added a privacy policy before we start our summer event. Going to get someone experienced to review this.

### CODE REVIEW

File: docs/privacy-policy.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+You have the right at any time to stop us from contacting you for marketing purposes.
+
+If you no longer wish to be contacted for marketing purposes, please unsubscribe at the bottom of one of our emails.
+
+## What are your data protection rights?
+
+We would like to make sure you are fully aware of your data protection rights. Every user is entitled to the following:
+
+**The right to access - You have the right to request us for copies of your personal data. We may charge a small admin fee associated with the cost of labour for this service.
```
Comment (jac0bwilson) [inline docs/privacy-policy.md]:
```suggestion
**The right to access** - You have the right to request us for copies of your personal data. We may charge a small admin fee associated with the cost of labour for this service.
```

File: docs/privacy-policy.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+- Privacy policies to other websites
+- Changes to our privacy policy
+- How to contact us
+- How to contact the appropriate authorities
+- What data do we collect?
+- We collect the following data:
+
+## What data do we collect?
+
+- Personal identification information (name, email address, phone number, - social media usernames)
```
Comment (jac0bwilson) [inline docs/privacy-policy.md]:
```suggestion
- Personal identification information (name, email address, phone number, social media usernames)
```

File: docs/privacy-policy.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+## How we will use your data?
+
+We collect your data so that we can:
+
+- Ensure the adequate safeguarding of minors engaging in our activities
+- Email you about relevant information pertaining to your engagement with - our activities
+- To review project submissions
+- To review talk and workshop applications
+- To schedule our activities
+- To enforce the [Hackathons for Schools Association Code of Conduct](/code-of-conduct)
```
Comment (jac0bwilson) [inline docs/privacy-policy.md]:
```suggestion
- To review project submissions
- Review talk and workshop applications
- Schedule our activities
- Enforce the [Hackathons for Schools Association Code of Conduct](/code-of-conduct)
```

File: docs/privacy-policy.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+## How we store your data?
+
+Data collected via Google Drive services, including Google Forms, are stored on William Russell's Google Drive account. This is only shared between the Hackathons for Schools Association Core Team.
+
+Due to a change in [Google's Terms and Service](https://policies.google.com/terms/update?hl=en-US), this data may be stored in either Europe or the United States of America.
+
+We may keep any data we collect about you for up to 12 months. Once this time period has expired we will delete your data.
+
+## Marketing
```
Comment (jac0bwilson) [inline docs/privacy-policy.md]: I'm not sure this should really be a possibility. Marketing to kids just feels wrong to me?

File: docs/privacy-policy.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+- Functionality - We use these cookies so that we recognise you on our websites and remember your previously selected preferences. They could include what language you prefer and location you are in. A mix of first-party and third-party cookies are used.
+
+
+## How to manage cookies
+
+You can set your browser not to accept cookies, and the above website tells you how to remove cookies from your browser. However, in a few cases, some of our website features may not function as a result.
+
+## Privacy policies of other websites
+
+- Our websites contain links to other websites. Our privacy policy applies only to our website, so if you click on a link to another website, you should read their privacy policy.
```
Comment (jac0bwilson) [inline docs/privacy-policy.md]:
```suggestion
Our websites contain links to other websites. Our privacy policy applies only to our website, so if you click on a link to another website, you should read their privacy policy.
```

**Review by jac0bwilson — APPROVED** [review]
Added some minor things, but overall this is pretty much done I reckon!

**Comment by jac0bwilson** [comment]
Would also suggest adding a link to the privacy policy in the `/docs/` page.

**Comment by wrussell1999** [comment]
Can you make these changes please! LGTM

File: docs/privacy-policy.md (hunk 4 above)
Comment (wrussell1999) [reply docs/privacy-policy.md]: Yeah agreed. Can you remove it please

**Pull request merged**